OrbitCaptureGgpService/main.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitCaptureGgpService.h"
+#include "OrbitVersion/OrbitVersion.h"
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "absl/flags/usage.h"
+#include "absl/flags/usage_config.h"
+
+ABSL_FLAG(uint16_t, grpc_port, 44767, "gRPC server port for capture ggp service");
+ABSL_FLAG(uint16_t, orbit_service_grpc_port, 44765, "gRPC server port for OrbitService");
+ABSL_FLAG(int32_t, pid, 0, "pid to capture");
+ABSL_FLAG(std::vector<std::string>, functions, {},
+          "Comma-separated list of functions to hook to the capture");
+ABSL_FLAG(std::string, file_name, "", "File name used for saving the capture");
+ABSL_FLAG(std::string, file_directory, "/var/game/",
+          "Path to locate orbit file. By default it is /var/game/");
```

Comment:
Nit: "orbit" -> ".orbit" to make it clear this is the ".orbit" capture file. (Or, if not, rename to "Orbit".)

Reply:
Makes sense, will update it as well; thanks!

Reply:
Done!